parm/config/config.base.emc.dyn

```diff
@@ -200,7 +200,7 @@ case "${APP}" in
         export DO_WAVE="YES"
         export WAVE_CDUMP="both"
         export cplwav2atm=".true."
-        export confignamevarfornems="${confignamevarfornems}_wave"
```

Comment:
Is this a name from UFS? I really dislike "out" instead of "outer", but if that is what UFS is doing, keep it for consistency.

Reply:
This is one of the names in UFS: https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/nems.configure.cpld_noaero_outwav.IN it's not completely consistent with what we're doing in UFS but similar.  This can be changed to outer or if you have a better name we can go with that too.  The name in UFS was not well liked, but no one could come up with a better name either.

Reply:
@WalterKolczynski-NOAA  any additional thoughts to this or leaving it as is for now?

Reply:
Yeah, let's use "outerwave"

Reply:
I'm in the middle of testing, so I'm going to continue that for now but will update this after I know the other is working.

Reply:
@WalterKolczynski-NOAA this has been updated